apps/stress-test/package.json

```diff
@@ -13,7 +13,6 @@
     "@fluentui/react": "*",
     "@fluentui/react-components": "*",
     "@fluentui/react-icons": "^2.0.237",
-    "@fluentui/web-components": "*",
```

Comment:
this needs to be verified and approved by @spmonahan, not sure if our stress test app is intendent to be run agains wc v3

Reply:
That's fine to remove. In fact, this has finally motivated me to remove the stress-test app as the current code lives at https://github.com/microsoft/tensile-perf

Reply:
It's still in master ? https://github.com/microsoft/fluentui/tree/master/apps/stress-test 

if you could send PR today that would be great !

Reply:
#31301

Reply:
Removal has been merged into master. Rebasing should remove this delta.